### ISSUE TITLE
Parsing destructuring assignment across lines throws error

### ISSUE THREAD

**Issue by travs** [ConsenSys/solidity-parser#74]
Example file:
```
pragma solidity ^0.4.0;              
contract Destructure {      

    function gimmeNumbers() returns (uint, uint, uint, uint) {                                       
        return (23, 88, 66, 0);                
    }                  
                       
    function destructureThem() {                     
        var (          
            num1, num2,                        
            num3, num4 
        ) = gimmeNumbers();  
    }                  
}  
```

Output:
```
Expected ",", comment, end of line, or whitespace but "n" found. Line: 10, Column: 13
```

**Comment by cgewecke** [comment]
It looks like I introduced this bug with #70. Whitespace should be consumed separately from the optional commas. I'll PR to change this shortly.

**Comment by federicobond** [comment]
Fixed by @cgewecke in #75.
